src/home_robot/home_robot/manipulation/grasping.py

```diff
@@ -0,0 +1,69 @@
+from typing import List, Optional, Tuple
+
+import numpy as np
+
+from home_robot.motion.stretch import HelloStretchIdx
+
+
+class SimpleGraspMotionPlanner(object):
+    """Simple top-down grasp motion planner for the Stretch."""
+
+    def __init__(self, robot):
```

Comment:
minor: typing

Because I was trying to look what `robot` is for trying this in sim. Do you think the robot client will need to be initialized in sim?